public_suffix_list.dat

```diff
@@ -13663,6 +13663,10 @@ london.cloudapps.digital
 // Submitted by <domeinnaam@minaz.nl>
 gov.nl
 
+// Grafana Labs https://grafana.com/
+// Submitted by <timur.olzhabayev@grafana.com>
```

Comment:
Please use a non-personal e-mail.

Reply:
Switched to non-personal e-mail and display name https://github.com/publicsuffix/list/pull/2188/commits/2ed0b3ed136957381a7d75790f1729e9185cc096